doc/api/stream.md

```diff
@@ -934,9 +939,8 @@ The Readable stream will properly handle multi-byte characters delivered through
 the stream that would otherwise become improperly decoded if simply pulled from
 the stream as `Buffer` objects.
 
-Encoding can be disabled by calling `readable.setEncoding(null)`. This approach
-is useful when working with binary data or with large multi-byte strings spread
-out over multiple chunks.
+Decoding can be disabled by calling `readable.setEncoding(null)` or
+`readable.setEncoding('buffer')`.
```

Comment:
It may just be me, but I had to read the docs for `readable.setEncoding()` several times to convince myself that it (and the change here) are correct due to the use of both encoding/decoding and encoded/decoded.